File: C/temp.c
#include <stdio.h>

int main() {
    float fahrenheit, celsius;

    printf("Insira a temperatura em Fahrenheit: ");
    scanf("%f", &fahrenheit);
    celsius = ((fahrenheit - 32) * 5) / 9;
    printf("A temperatura em Celsius é: %.2f", celsius);

    return 0;
}

File: Go/temp.go
package main

import (
	"fmt"
)

func main() {
	var fahrenheit float64
	fmt.Print("Insira a temperatura em fahrenheit: ")
	fmt.Scan(&fahrenheit)
	celsius := ((fahrenheit - 32) * 5) / 9
	fmt.Printf("A temperatura em celsius é: %.2f\n", celsius)
}
